commands/proposal_submission.go

```diff
@@ -975,6 +977,27 @@ func checkPriceMonitoring(parameters *protoTypes.PriceMonitoringParameters, pare
 	return errs
 }
 
+func checkLiquidationStrategy(params *protoTypes.LiquidationStrategy, parent string) Errors {
```

Comment:
Is there no validation required fro:   `uint64 full_disposal_size = 3;`

Reply:
No, the spec says that the range of this parameter is 0-max, so no validation required.